Modules/SubModule_Import/xASL_imp_DCM2NII_Subject_ShuffleTheDynamics.m

```diff
@@ -159,7 +159,7 @@
 
     if xASL_exist(x.dir.studyPar,'file')
```

Comment:
For a json file, `exist` works as well as `xASL_exist`

Reply:
Let's not change this too much, because it was already modified/renamed etc. in another PR. Keep it like it is for now 👍

Reply:
👍